### PR TITLE
Install python3 and python3-yaml as global dependencies

### PR DESCRIPTION
**Pull request by Darchiv**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -79,6 +79,7 @@ RUN apt-get update && \
         vim \
         xxd \
         cmake \
+        python3-yaml \
```
Comment (nalajcie) [inline build/Dockerfile:82]: maybe add also comment above regarding why it's needed (eg. ` python3-yaml: phoenix-rtos-build tools`)

File: gh-runner/Dockerfile [hunk 2]
```diff
@@ -82,6 +82,7 @@ RUN apt-get update && \
         libhidapi-dev \
         python3-dev \
         python3-pip \
+        python3-yaml \
```
Comment (nalajcie) [inline gh-runner/Dockerfile:85]: probably not needed here as we're not building anything on self-hosted github runners (just running HW tests - using trunner in venv).

File: build/Dockerfile (hunk 1 above)
Comment (Darchiv) [reply build/Dockerfile:84]: Done. Also added python3 package just in case

File: gh-runner/Dockerfile (hunk 2 above)
Comment (Darchiv) [reply gh-runner/Dockerfile]: Right, it seems to not be needed. Removed

File: build/Dockerfile (hunk 1 above)
Comment (nalajcie) [reply build/Dockerfile:84]: in the comment above please - where all of the other deps are described:
image

**Review by nalajcie — APPROVED** [review]

**Pull request merged**